src/builders/polylines.js

```diff
@@ -40,579 +43,444 @@ export function buildPolylines (
         miter_limit
     }) {
 
-    var cornersOnCap = corners_for_cap[cap] || 0;         // default 'butt'
-    var trianglesOnJoin = triangles_for_join[join] || 0;  // default 'miter'
+    var cap_type = cap ? CAP_TYPE[cap.toUpperCase()] : CAP_TYPE.BUTT;
```

Comment:
Can do. You're right, it was just for a stylistic uppercase reason that I personally like for constants, but shouldn't come with a performance cost.